app/src/main/java/pl/szczodrzynski/edziennik/utils/SwipeRefreshLayoutNoIndicator.java

```diff
@@ -83,7 +83,6 @@ public SwipeRefreshLayoutNoIndicator(@NonNull Context context, @Nullable Attribu
 
     @Override
     public boolean onInterceptTouchEvent(MotionEvent ev) {
-        ev.setSource(0x10000000);
```

Comment:
This source is used later. Please wrap this in a check that disables this behavior for API 32+ only.